concept/concept.md

```diff
@@ -0,0 +1,254 @@
+# Network Concept
+
+Human Connection is a nonprofit social, action and knowledge network that connects information to action and promotes positive local and global change in all areas of life. 
+
+- **Social**: Interact with other people not just by commenting their posts, but by providing **Pro & Contra** arguments, give a **Versus** or ask them by integrated **Chat** or **Let's Talk**
+
+- **Knowledge**: Read articles about interesting topics and find related posts in the **More Info** tab or by **Filtering** based on **Categories** and **Tagging** or by using the **Fulltext Search**.
+
+- **Action**: Don't just read about how to make the world a better place, but come into **Action** by following provided suggestions on the **Action** tab provided by other people or **Organisations**.
+
+## Features
+
+The following features will be implemented. This gets done in three steps: 
+
+1. First we will implement a basic feature set and provide an test system for to test the basic network functionality. 
+
+2. In a second step we will go live into the public with an advanced feature set which includes also all the functionality in code and organisation that will be needed to drive a bigger public social network.
+
+3. In a third step all the remaining Features will be implemented to build the full product. 
```

Comment:
```suggestion
3. In a third step all the remaining features will be implemented to build the full product. 
```
Will there ever be a "finished" product? :thinking: :wink:

Reply:
Corrected in updated branch network-concept